server/middleware.go

```diff
@@ -19,9 +19,9 @@ func softMiddleware(ac *appCfg.Config) wish.Middleware {
 				if active {
 					return
 				}
-				ctx := s.Context()
-				ctx = context.WithValue(ctx, "config", ac) //nolint:revive
-				ctx = context.WithValue(ctx, "session", s) //nolint:revive
+				sctx := s.Context()
+				ctx := context.WithValue(sctx, "config", ac) //nolint:revive
+				ctx = context.WithValue(sctx, "session", s)  //nolint:revive
```

Comment:
Just curious why this needs / should be renamed to `sctx`? It looks like the `ctx` will just get overridden and not contain the `"config", ac` value.

Reply:
It's because of gliderlabs/ssh v0.3.4 release commit https://github.com/gliderlabs/ssh/commit/dd71f10d4d7bd8fbc2dac2ea8c28ae267f26dbbd

`s.Context()` now has type `ssh.Context` and assignment to type `context.Context` gives errors. I guess something like this would look better
```
				var ctx context.Context = s.Context()
				ctx = context.WithValue(ctx, "config", ac) //nolint:revive
				ctx = context.WithValue(ctx, "session", s) //nolint:revive
```

Reply:
Ah, gotcha. LGTM, then!